tests/pytorch/iris.py

```diff
@@ -6,23 +6,32 @@
 import torch
 from torch import nn
 import torch.nn.functional as F
+from packaging.version import Version
 
-# NB: Older versions of PyTorch Lightning define native APIs for metric computation,
-# (e.g., pytorch_lightning.metrics.Accuracy), while newer versions rely on the `torchmetrics`
-# package (e.g. `torchmetrics.Accuracy)
-try:
-    from torchmetrics import Accuracy
-except ImportError:
-    from pytorch_lightning.metrics import Accuracy
```

Comment:
Do we need to fix this block?

Reply:
Can we just define 

```python
def create_multiclass_accuracy()
    if Version(torchmetrics.__version__) >= Version("0.11"):
        return Accuracy(task="multiclass")
    else:
        return Accuracy()
```

Reply:
I moved it into `create_multiclass_accuracy` , shall it be put in top level ?

Reply:
```python
def create_multiclass_accuracy():
    # NB: Older versions of PyTorch Lightning define native APIs for metric computation,
    # (e.g., pytorch_lightning.metrics.Accuracy), while newer versions rely on the `torchmetrics`
    # package (e.g. `torchmetrics.Accuracy)
    try:
        from torchmetrics import Accuracy
        import torchmetrics
        if Version(torchmetrics.__version__) >= Version("0.11"):
            return Accuracy(task="multiclass")
        else:
            return Accuracy()
    except ImportError:
        from pytorch_lightning.metrics import Accuracy
        return Accuracy()
```

vs.

```python
try:
    from torchmetrics import Accuracy
except ImportError:
    from pytorch_lightning.metrics import Accuracy

def create_multiclass_accuracy()
    if Version(torchmetrics.__version__) >= Version("0.11"):
        return Accuracy(task="multiclass")
    else:
        return Accuracy()
```

Reply:
To me, the second one looks cleaner.

Reply:
Does the second one work?

Reply:
Perhaps the first one might be less confusing.